backup/queries_relations.go

```diff
@@ -126,6 +126,8 @@ func getUserTableRelations(connectionPool *dbconn.DBConn) []options.Relation {
 			WHERE c.relispartition = true OR c.relkind = 'p'
 			GROUP BY 1
 		) AS prt ON prt.oid = c.oid`
+		// Filter out partitions whose root is in extension.
+		childPartitionFilter = `AND (c.relispartition IS false or pg_partition_root(c.oid) NOT IN (SELECT objid FROM pg_depend WHERE deptype = 'e'))`
```

Comment:
> pg_partition_root(c.oid) is called above in prt var, why not use it?

prt.id it will not be NULL only for root partitions since we are doing group by on this column. Therefore, we will filter out all the partitions (which we do not want to do for 7X).

Reply:
```suggestion
		childPartitionFilter = `AND NOT (c.relispartition AND pg_partition_root(c.oid) IN (SELECT objid FROM pg_depend WHERE deptype = 'e'))`
```